package.json

```diff
@@ -15,8 +15,8 @@
     "production": "npm-run-all build:*"
   },
   "devDependencies": {
-    "@openeuropa/bcl-builder": "1.6.0",
-    "@openeuropa/bcl-theme-default": "1.6.0",
+    "@openeuropa/bcl-builder": "0.3421.2024101161200",
+    "@openeuropa/bcl-theme-default": "0.3421.2024101161200",
```

Comment:
testing was requested, the ticket asked for "copyright page" but that's a mistake as its part of an epic and not merged yet, so we can use  the patterns page where there is a modal and 2 overlays available